flent/runners.py

```diff
@@ -1108,7 +1108,7 @@ def check(self):
             args['marking'] = self.parse_marking(args['marking'], "-Y {}", True)
 
         if args['cong_control']:
-            args['cong_control'] = "-K {0}".format(args['cong_control'])
+            args['cong_control'] = "-K {0},{0}".format(args['cong_control'])
```

Comment:
Netperf already sets the same argument for both directions if there's no comma in the value, so this is not needed...

Reply:
You are right.  I'll remove that.  Thanks for the catch.